CHANGELOG.md

```diff
@@ -1,10 +1,9 @@
 # Changelog
 All notable changes to this project since 0.82.0 will be documented in this file.
 
-## [2.2.0] - 2021-01-06
-### Changed
-- Remove explicit cryptography package install to resolve dependency conflicts so tests can run
-- Add local registry support for Windows
+## [3.0.0] - 2021-01-04
+- Remove support for Python2. Only Python 3.6 and 3.8 are currently supported
```

Comment:
Only python 3.6 and 3.8? Why not Python 3.7?

Reply:
Woops, typo. Supposed to be 3.7 and 3.8